ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewManagersPropertyCache.java

```diff
@@ -45,13 +45,11 @@ public static void clear() {
     protected final Method mSetter;
     protected final @Nullable Integer mIndex; /* non-null only for group setters */
 
-    // The following Object arrays are used to prevent extra allocations from varargs when we call
```

Comment:
Do you mind keeping the original comment here for the posterity? Optimizations like these can be very confusing to figure out without explanation.
Feel free to add ThreadLocal comment under original though :)